Frontend/src/pages/crowdactions/Create/form.ts

```diff
@@ -84,6 +86,7 @@ export const validations = Yup.object({
   hashtags: Yup.string()
     .max(30, 'Please keep the number of hashtags civil, no more then 30 characters')
     .matches(/^[a-zA-Z_0-9]+(;[a-zA-Z_0-9]+)*$/, 'Don\'t use spaces or #, must contain a letter, can contain digits and underscores. Separate multiple tags with a colon \';\''),
+  instagramUser: Yup.string(),
```

Comment:
Min / Max length check like in back end is missing

Reply:
A minimum check is not necessary. If the string is empty, it will send a null, which is also valid on the backend. I added the max check though.